package.json

```diff
@@ -25,7 +25,9 @@
     "prepare": "patch-package",
     "prettier-project": "prettier --write 'src/**/*.{js,ts,tsx,md,graphql}'",
     "schema-drift": "babel-node --no-warnings --extensions '.ts,.js' ./scripts/schema-drift.ts",
+    "setup": " ./scripts/setup.sh",
```

Comment:
I like this!
Will add to force tomorrow along with `sync-env` and check to test is shared config is out of date, unless you beat me to it :-)

Reply:
Joey brought up a good point about missing `yarn` 🤦